src/Admin/AbstractAdmin.php

```diff
@@ -1259,7 +1259,7 @@ final public function getFilterFieldDescriptions(): array
         return $this->filterFieldDescriptions;
     }
 
-    final public function addChild(AdminInterface $child, string $field): void
+    final public function addChild(AdminInterface $child, ?string $field): void
```

Comment:
```suggestion
    final public function addChild(AdminInterface $child, ?string $field = null): void
```

Isnt this more correct?